app/main.py

```diff
@@ -1 +1,42 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
You can use `set` for to leverage faster membership checks and removals.

Reply:
I can`t, because in test file i have this test:

![image](https://github.com/mate-academy/py-herbivores-and-carnivores/assets/168102872/04af8c30-87fd-4480-80d0-a5a327bb1657)

Reply:
and more other checks where we need to use list

Reply:
ok